examples/atmega2560/src/bin/atmega2560-blink.rs

```diff
@@ -0,0 +1,21 @@
+#![no_std]
+#![no_main]
+
+use panic_halt as _;
+use atmega_hal::clock;
+use atmega_hal::delay::Delay;
+use embedded_hal::delay::DelayNs;
+
+#[avr_device::entry]
+fn main() -> ! {
+    let dp = atmega_hal::Peripherals::take().unwrap();
+    let pins = atmega_hal::pins!(dp);
+    let mut delay = Delay::<clock::MHz16>::new();
```

Comment:
Just a though:  We could use one of the examples to show off a nicer way for dealing with delays:  Using the previously mentioned `CoreClock`, you can define more things:

```rust
type CoreClock = atmega_hal::clock::MHz16;

type Delay = atmega_hal::delay::Delay<crate::CoreClock>;

fn delay_ms(ms: u16) {
    crate::Delay::new().delay_ms(u32::from(ms))
}

fn delay_us(us: u32) {
    crate::Delay::new().delay_us(us)
}
```

Reply:
Yes, I saw this one in the Arduino HAL implementation. Will update blink example.